src/components/app/list-new/AppList.tsx

```diff
@@ -46,8 +47,8 @@ export default function AppList({ isSuperAdmin, appListCount, isArgoInstalled }:
     const [syncListData, setSyncListData] = useState<boolean>()
 
     // API master data
-    const [projectListRes, setProjectListRes] = useState({ result: [] })
-    const [environmentListRes, setEnvironmentListRes] = useState({ result: [] })
+    const [projectListRes, setProjectListRes] = useState([])
+    const [environmentClusterListRes, setEnvironmentClusterListRes] = useState<EnvironmentClusterList>()
```

Comment:
Aren't we initializing with empty array?

Reply:
Not required